NAMESPACE

```diff
@@ -1,5 +1,7 @@
 # Generated by roxygen2 (4.1.1): do not edit by hand
 
+export(gdp_count_time_steps)
+export(gdp_ids_and_units)
 export(write_point_dsf)
-export(write_timeseries_dsf)
+export(write_timeseries_dsg)
```

Comment:
Discrete Sampling Geometries is the name of this class of things in the NetCDF-CF spec.